### PR TITLE
nrf/drivers: Adding license text to ticker.h and softpwm.h.

### PR DESCRIPTION
**Pull request by glennrub**

### CODE REVIEW

**Comment by glennrub** [comment]
@dpgeorge, could you take a look at this?
If you approve, I can either merge this on top of master, or you could cherry-pick this commit into to new Nordic PR in mainline.

File: ports/nrf/drivers/softpwm.h [hunk 1]
```diff
@@ -1,3 +1,40 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2013-2016 The MicroPython-on-micro:bit Developers, as listed
+ * in the accompanying AUTHORS file
```
Comment (dpgeorge) [inline ports/nrf/drivers/softpwm.h]: I would have just written this as:

Copyright (c) 2013-2016 The MicroPython-on-micro:bit Developers: Damien P. George (@dpgeorge),
Nicholas H. Tollervey ...

Just to keep it compact, but still honest.

**Comment by glennrub** [comment]
Updated, and force-pushed.

**Comment by dpgeorge** [comment]
Thanks.  I think this is a fair and correct way to do it.

**Comment by glennrub** [comment]
I'll merge this on top of tralamazza master just to get this patch into this source as well. I hope that makes sense. I guess it should be fairly simple to do a cherry pick of this commit into your PR branch. Right?

**Comment by dpgeorge** [comment]
> I guess it should be fairly simple to do a cherry pick of this commit into your PR branch.

Yes, no problem.

**Pull request merged**